### PR TITLE
Use json_decode for effects data

### PR DESCRIPTION
**Pull request by TheDiamondYT1**
## Introduction

Matches [this commit](https://github.com/pmmp/PocketMine-MP/commit/fe4322bea957b422ea8596712ccaf022a388c9b7#diff-c0f9e4f47d9a1ae7abba9ce1d681fd89R86) (OCD)

### CODE REVIEW

File: src/pocketmine/entity/Effect.php [hunk 1]
```diff
@@ -60,9 +59,12 @@ class Effect{
...
 
 	public static function init(){
-		$config = new Config(\pocketmine\PATH . "src/pocketmine/resources/effects.json", Config::JSON, []);
+		$data = json_decode(file_get_contents(\pocketmine\PATH . "src/pocketmine/resources/effects.json"), true);
+		if(!is_array($data)){
+			throw new \RuntimeException("Effects data could not be read");
+		}
 
-		foreach($config->getAll() as $name => $data){
+		foreach($data as $name => $data){
```
Comment (dktapps) [inline src/pocketmine/entity/Effect.php]: Bad move.

**Review by dktapps — CHANGES_REQUESTED** [review]

File: src/pocketmine/entity/Effect.php (hunk 1 above)
Comment (dktapps) [inline src/pocketmine/entity/Effect.php]: (lucky PhpStorm spotted this, I almost merged this without looking >_>)

Comment (TheDiamondYT1) [inline src/pocketmine/entity/Effect.php]: What is wrong?

Comment (TheDiamondYT1) [inline src/pocketmine/entity/Effect.php]: Oh..im so dumb

**Comment by TheDiamondYT1** [comment]
Sorry about that @dktapps. Im actually using IntelliJ... :rofl:

**Review by dktapps — APPROVED** [review]

**Pull request merged**